sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -185,34 +140,63 @@ public class AADOAuth2LoginConfigSample extends AADWebSecurityConfigurerAdapter
     }
 }
 ```
+   
+#### Support the use of `@RegisteredOAuth2AuthorizedClient` to get `OAuth2AuthorizedClient`:
```

Comment:
This should under `#### Web application visit other resource servers`?

Reply:
And `#### Standalone web application` should be the first scenario.
So the order should be like this:
```
Standalone web application
Web application visit other resource servers
On-demand authorization
```